### PR TITLE
Nameres fixes

### PR DESCRIPTION
**Pull request by Roldak**

### CODE REVIEW

File: ada/testsuite/tests/c_api/unit_file_providers/test.out [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Calling ufp_get_file_from_name (some_field=42, kind=0) with name: "bar"
 Calling ufp_get_file_from_name (some_field=42, kind=0) with name: "foo"
+Calling ufp_get_file_from_name (some_field=42, kind=0) with name: "bar"
```
Comment (Roldak) [inline ada/testsuite/tests/c_api/unit_file_providers/test.out]: I have no idea about this diff, is it okay?

Comment (pmderodat) [reply ada/testsuite/tests/c_api/unit_file_providers/test.out]: Probably. This test exposes internal implementation details that are not really something we guarantee, so it’s been updated several times already. As long as you understand why it changed…

File: ada/testsuite/tests/name_resolution/pragma_assert/p.adb [hunk 2]
```diff
@@ -1,8 +1,15 @@
-   procedure P is
-      function "-"(I: Integer) return Integer is (I*I);
+procedure P is
+   function "-" (I: Integer) return Integer is (I*I);
```
Comment (raph-amiard) [inline ada/testsuite/tests/name_resolution/pragma_assert/p.adb]: Look at you being all fancy about Ada coding style

File: ada/testsuite/tests/c_api/unit_file_providers/test.out (hunk 1 above)
Comment (raph-amiard) [reply ada/testsuite/tests/c_api/unit_file_providers/test.out]: This test is kind of bothersome, but not enough for us to fix it I'd say ...

File: ada/testsuite/tests/name_resolution/pragma_assert/p.adb (hunk 2 above)
Comment (pmderodat) [reply ada/testsuite/tests/name_resolution/pragma_assert/p.adb]: Nice :) We just miss spaces around the `*` operator :-)

Comment (Roldak) [reply ada/testsuite/tests/name_resolution/pragma_assert/p.adb]: Ha, too late. This will be for the next person who touches this test!

**Pull request merged**